src/main/java/seedu/address/model/person/UniquePersonList.java

```diff
@@ -134,4 +135,44 @@ private boolean personsAreUnique(List<Person> persons) {
         }
         return true;
     }
+
+    /**
+     * Sorts the list by given {@code category}.
+     * @param category
+     */
+    public void sort(String category) {
+        if (category.equals("name")) {
```

Comment:
Perhaps consider declaring constants instead, to avoid magic strings, as mentioned elsewhere?